### ISSUE TITLE
Copy-PowerBIReportContentToBlankPBIXFile doesn't work for reports in My Workspace

### ISSUE THREAD

**Issue by rudzimj** [JamesDBartlett3/PowerBits#1]
There is no ID for My Workspace. To make it work, I changed $true to $false here:
```
    [parameter(Mandatory = $false)][string]$sourceWorkspaceId,
```
And added this logic:
```
    if ($targetWorkspaceId -eq "") {
      $endpointUri = "$pbiApiBaseUri/reports/$targetReportId/UpdateReportContent"
    } else {
      $endpointUri = "$pbiApiBaseUri/groups/$targetWorkspaceId/reports/$targetReportId/UpdateReportContent"
    }
```

It would also be nice to add a sample to invoke the function:
```
#For My Workspace, no need to provide WorkspaceID
Copy-PowerBIReportContentToBlankPBIXFile -sourceReportId 6a9ffeff-5d4f-4526-bfc9-2b77fc756f71 -targetReportId 32b273ff-b40f-48ff-86ea-20c0e2c55cc1
```

**Comment by JamesDBartlett3** [comment]
Cool idea. ~~Feel free to submit a PR, and I'll merge it.~~

_Update: See comment below._

**Comment by JamesDBartlett3** [comment]
@rudzimj, I just discovered that Microsoft has finally rolled out native support for downloading reports originally authored in the Power BI service.

![image](https://user-images.githubusercontent.com/37491308/196057498-43a0ba91-0261-4f10-b46c-8981a9cb738a.png)

Therefore, this function is no longer strictly necessary, and as such, I won't be making any future modifications to it. For now, I've decided to keep it here, so it can serve as a teaching aid, to refresh my own memory, and of course, as training data for GitHub Copilot! 😉 

Thank you for bringing my attention to this issue, and I hope you'll continue contributing to this repository going forward.

~ James
